### PR TITLE
DOCS-2128: Publishes Calico Enterprise 3.17.4

### PR DESCRIPTION
**Pull request by ctauchen**

### CODE REVIEW

**Comment by ctauchen** [comment]
Cc @danudey @rene-dekker

File: calico-enterprise_versioned_docs/version-3.17/releases.json [hunk 1]
```diff
@@ -1,4 +1,271 @@
 [
+  {
+    "title": "v3.17.4",
+    "tigera-operator": {
+      "image": "tigera/operator",
+      "version": "v1.30.8",
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.17/releases.json]: Likely to be v1.30.10, as 8&9 already exist: https://github.com/tigera/operator/releases

File: calico-enterprise_versioned_docs/version-3.17/releases.json [hunk 2]
```diff
@@ -1,4 +1,271 @@
...
+        "image": "tigera/honeypod-exp-service",
+        "version": "v3.17.4"
+      },
+      "honeypod-controller": {
+        "image": "tigera/honeypod-controller",
+        "version": "v3.17.4"
+      },
+      "key-cert-provisioner": {
+        "image": "tigera/key-cert-provisioner",
+        "version": "v1.1.14",
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.17/releases.json]: We need to cut a new v1.1.19

File: calico-enterprise_versioned_docs/version-3.17/releases.json [hunk 3]
```diff
@@ -1,4 +1,271 @@
...
+      "fluentd-windows": {
+        "image": "tigera/fluentd-windows",
+        "version": "v3.17.4"
+      },
+      "es-proxy": {
+        "image": "tigera/es-proxy",
+        "version": "v3.17.4"
+      },
+      "eck-kibana": {
+        "version": "7.17.14"
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.17/releases.json]: 7.17.18

File: calico-enterprise_versioned_docs/version-3.17/releases.json [hunk 4]
```diff
@@ -1,4 +1,271 @@
...
+      },
+      "kibana": {
+        "image": "tigera/kibana",
+        "version": "v3.17.4"
+      },
+      "eck-elasticsearch": {
+        "version": "7.17.14"
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.17/releases.json]: 7.17.18

File: calico-enterprise_versioned_docs/version-3.17/_includes/release-notes/_v3.17.4-release-notes.mdx [hunk 5]
```diff
@@ -0,0 +1,9 @@
+DD March 2024
+
+### Improvements
+-
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.17/_includes/release-notes/_v3.17.4-release-notes.mdx]: - Decreases the validity of JWTs issued by Dex to 15m (down from 24h)
- Adds a configurable option for `priorityClassName` to the `EgressGateWay` CRD
@mazdakn @sridhartigera Is this ^ ok?

Comment (rene-dekker) [reply calico-enterprise_versioned_docs/version-3.17/_includes/release-notes/_v3.17.4-release-notes.mdx]: - Policy recommendation excludes Openshift namespaces by default.

Comment (rene-dekker) [reply calico-enterprise_versioned_docs/version-3.17/_includes/release-notes/_v3.17.4-release-notes.mdx:4]: - If kube-controllers metric port is set to 0, no ingress rule will be created.

File: calico-enterprise_versioned_docs/version-3.17/_includes/release-notes/_v3.17.4-release-notes.mdx [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+
+### Bug fixes
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.17/_includes/release-notes/_v3.17.4-release-notes.mdx:9]: - If kube-controllers metric port is set to 0, no ingress rule will be created.

Comment (vara2504) [reply calico-enterprise_versioned_docs/version-3.17/_includes/release-notes/_v3.17.4-release-notes.mdx:9]: Has bug fix for:  Kubernetes audit logs for EKS does not work and pods get stuck with this error: eks-log-forwarder-startup time="2024-01-23T17:42:29Z" level=fatal msg="Error setting up elastic client.

Comment (rene-dekker) [reply calico-enterprise_versioned_docs/version-3.17/_includes/release-notes/_v3.17.4-release-notes.mdx:9]: @vara2504 How about: "Fixed a bug introduced in v3.17.3 that stopped the eks-log-forwarder deployment from starting."

File: calico-enterprise_versioned_docs/version-3.17/releases.json (hunk 2 above)
Comment (danudey) [reply calico-enterprise_versioned_docs/version-3.17/releases.json]: v1.1.19 has been cut

File: calico-enterprise_versioned_docs/version-3.17/releases.json (hunk 1 above)
Comment (ctauchen) [reply calico-enterprise_versioned_docs/version-3.17/releases.json]: Done.

File: calico-enterprise_versioned_docs/version-3.17/releases.json (hunk 2 above)
Comment (ctauchen) [reply calico-enterprise_versioned_docs/version-3.17/releases.json]: Done.

File: calico-enterprise_versioned_docs/version-3.17/releases.json (hunk 3 above)
Comment (ctauchen) [reply calico-enterprise_versioned_docs/version-3.17/releases.json]: Done.

File: calico-enterprise_versioned_docs/version-3.17/releases.json (hunk 4 above)
Comment (ctauchen) [reply calico-enterprise_versioned_docs/version-3.17/releases.json]: Done.

File: calico-enterprise_versioned_docs/version-3.17/_includes/release-notes/_v3.17.4-release-notes.mdx (hunk 5 above)
Comment (ctauchen) [reply calico-enterprise_versioned_docs/version-3.17/_includes/release-notes/_v3.17.4-release-notes.mdx]: Added all.

File: calico-enterprise_versioned_docs/version-3.17/_includes/release-notes/_v3.17.4-release-notes.mdx (hunk 6 above)
Comment (ctauchen) [reply calico-enterprise_versioned_docs/version-3.17/_includes/release-notes/_v3.17.4-release-notes.mdx:9]: Added.

**Review by rene-dekker — APPROVED** [review]
lgtm

**Pull request merged**